### DISCUSSION TITLE
loopback 4 connector for Elastic Search

### DISCUSSION THREAD

**Discussion by fidgi** [loopbackio/loopback-next#8573] [Q&A]
Hi,

Is there a maintained and up-to-date loopback 4 connector for Elastic Search ?

This one https://github.com/loopbackio/loopback-connector-elastic-search seems a bit outdated .

It still depends on the legacy Elastic Search Javascript client and there was no release since 2 years.

Regards,

**Comment by achrinza** [comment]
It's something that unfortunately we haven't had the chance to work on yet.

I've opened https://github.com/loopbackio/loopback-connector-elastic-search/issues/146 to track getting the connector up to speed with the current landscape.
